src/controller/python/ChipDeviceController-ScriptDevicePairingDelegate.cpp

```diff
@@ -180,5 +185,40 @@ ScriptDevicePairingDelegate::GetOpenWindowCallback(Controller::CommissioningWind
     return &mOpenWindowCallback;
 }
 
+void ScriptDevicePairingDelegate::OnICDRegistrationComplete(NodeId nodeId, uint32_t icdCounter)
```

Comment:
Is it possible to pass the fabric index as argument, rather than relying on state?